### PR TITLE
Refine Kubectl GitBook

### PR DESCRIPTION
**Pull request by pwittrock**
Flesh out the GitBook more.

Hosted at: https://pwittrock-kubectl.firebaseapp.com/

### CODE REVIEW

File: docs/book/pages/dam_apply.md [hunk 1]
```diff
@@ -19,31 +23,64 @@ Apply will update a Kubernetes cluster to match state defined locally in files.
 - **Resource Config**: *Files* declaring the desired state for Resources - e.g. deployment.yaml.
   Resources are created and updated using Apply with these files.
 
-*kubectl apply* Creates, Updates and Deletes Resources using Resource Config with a *apply.yaml*.
+*kubectl apply* Creates and Updates Resources through local or remote files.  This may be through
+either raw Resource Config or *kustomization.yaml*.
```
Comment (DanyC97) [inline docs/book/pages/dam_apply.md]: will be good to maybe expand and provide more info about kustomization. Not everyone managed to read the [blog](https://kubernetes.io/blog/2018/05/29/introducing-kustomize-template-free-configuration-customization-for-kubernetes/) you wrote or git repo [workflow](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/workflows.md)

**Comment by Liujingfang1** [comment]
/lgtm
The deploy/netlify failed since the product branch is set to book branch. I'll update it.
Not sure why the travis-ci failed.

**Pull request merged**